DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.schema.mof

```diff
@@ -2,8 +2,10 @@
 class MSFT_xDSCWebService : OMI_BaseResource
 {
   [Key] string EndpointName;
-  [required, Description("Can take the value AllowUnencryptedTraffic for setting up a non SSL based endpoint")]
+  [write , Description("Can take the value AllowUnencryptedTraffic for setting up a non SSL based endpoint")]

```

Comment:
We still need some mechanism to require the user to explicitly opt-out of being secure